src/app/services/code-view/angular-code-service.ts

```diff
@@ -48,11 +53,29 @@ export class AngularCodeService extends CodeService {
                     this.getAngularSampleFiles(baseUrl, codeViewsData, () => this.renderFooters(codeViewsData));
                 }
             }
-            if (!(util.isIE || util.isEdge)) {
-                $standaloneliveEditingButtons.on('click', this.onAngularGithubProjectStandaloneButtonClicked());
-            } else {
-                $standaloneliveEditingButtons.css("display", "none");
+        }
+    }
+
+    private genetareStandaloneButtonsApp($buttons: any) {
+        if (!(util.isIE || util.isEdge)) {
+            $.each($buttons, (index, element) => {
+                let $button = $(element);
+                let samplesBaseUrl = $button.attr(this.demosBaseUrlAttrName)!;
+                let sampleUrl = $button.attr(this.buttonSampleSourceAttrName)!;
+                if (!this.demosUrls.has(samplesBaseUrl)) {
+                    this.demosUrls.set(samplesBaseUrl, [{ url: sampleUrl, element: $button }]);
+                } else {
+                    this.demosUrls.get(samplesBaseUrl)!.push({ url: sampleUrl, element: $button });
+                }
+            });
+
+            let allDemosBaseUrls = this.demosUrls.keys();
+            for (const baseUrl of allDemosBaseUrls) {
+                let buttonsData = this.demosUrls.get(baseUrl)!;
+                this.generateLiveEditingAngularApp(baseUrl, buttonsData);
             }
+        } else {
+            $buttons.css("display", "none");
         }
 
         this.demosUrls.clear();
```

Comment:
The demosUrls and the sharedfilecontent will be cleared only if there are standalone buttons.
Also the sampleFilesContentByUrl must also be emptied